cmd/podman/images/sign.go

```diff
@@ -47,6 +47,7 @@ func init() {
 	certDirFlagName := "cert-dir"
 	flags.StringVar(&signOptions.CertDir, certDirFlagName, "", "`Pathname` of a directory containing TLS certificates and keys")
 	_ = signCommand.RegisterFlagCompletionFunc(certDirFlagName, completion.AutocompleteDefault)
+	flags.BoolVarP(&signOptions.All, "all", "a", false, "Allow signing all the manifest of the image manifest list")
```

Comment:
“Allow” seems unnecessary.

Maybe say “multi-architecture image” instead of the implementation jargon.

(Similarly in the man page.)